packages/form-js-editor/package.json

```diff
@@ -20,9 +19,12 @@
   "types": "dist/types/index.d.ts",
   "scripts": {
     "all": "run-s lint test build",
-    "build": "run-p bundle generate-types",
+    "build": "run-p bundle bundle:scss generate-types",
     "bundle": "rollup -c --failAfterWarnings --bundleConfigAsCjs",
-    "bundle:watch": "rollup -c -w --bundleConfigAsCjs",
+    "bundle:scss": "sass --no-source-map --load-path=\"../../node_modules\" assets/index.scss dist/assets/form-js-editor.css",
```

Comment:
I decided to use this simple bundling step instead of a rollup plugin. Rationale: the rollup plugins I used (https://github.com/thgh/rollup-plugin-scss, https://github.com/egoist/rollup-plugin-postcss) didn't work as expected.